install_config/storage_examples/ceph_example.adoc

```diff
@@ -273,3 +273,44 @@ spec:
 <1> `*securityContext*` must be defined at the pod level, not under a specific container.
 <2> All containers in the pod will have the same `*fsGroup*` ID.
 ====
+
+[[using-ceph-rbd-setting-default-secret]]
+== Setting ceph-user-secret as Default for Projects
+If you would like to make the persistant storage available to every project you have to modify the default project template.
+You can read more on modifying the default project template here link:https://docs.openshift.org/latest/admin_guide/managing_projects.html#selfprovisioning-projects[Self-provisioning Projects].
+Adding this to your default project template will allow every user who has access to create a project access to the ceph cluster. 
+
+.Default Project Example
+====
+[source,yaml]
+----
+...
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: project-request
+objects:
+- apiVersion: v1
+  kind: Project
+  metadata:
+    annotations:
+      openshift.io/description: ${PROJECT_DESCRIPTION}
+      openshift.io/display-name: ${PROJECT_DISPLAYNAME}
+      openshift.io/requester: ${PROJECT_REQUESTING_USER}
+    creationTimestamp: null
+    name: ${PROJECT_NAME}
+  spec: {}
+  status: {}
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: ceph-user-secret
+  data:
+    key: yoursupersecretbase64keygoeshere <1>
+  type:
+    kubernetes.io/rbd
+...
+----
+<1> Place your super secret ceph user key here in base64 format. See link:https://docs.openshift.org/latest/install_config/storage_examples/ceph_example.html#using-ceph-rbd-creating-the-ceph-secret[Creating the Ceph Secret]
```

Comment:
s/ceph/Ceph

See xref:../install_config/storage_examples/ceph_example.adoc#using-ceph-rbd-creating-the-ceph-secret[Creating the Ceph Secret].